CorsixTH/Lua/utility.lua

```diff
@@ -216,6 +216,12 @@ DrawFlags.ListBottom      = 2^11
 DrawFlags.BoundBoxHitTest = 2^12
 DrawFlags.Crop            = 2^13
 
+-- Keep in sync with animation_effect in th_gfx_common.h
+AnimationEffect = {}
+AnimationEffect.None = 0
+AnimationEffect.Glowing = 1
+AnimationEffect.Jelly = 2
```

Comment:
Add an end of line comment e.g. not yet implemented. Just incase jelly isn't implemented straight away